### DISCUSSION TITLE
Should M1/ARM-based "brew doctor" be checking files in /usr/local

### DISCUSSION THREAD

**Discussion by jimtut** [Homebrew/discussions#1468] [Tap maintenance and Homebrew development] [answered]
I ran `brew doctor` recently to find and clean some legit problems (unlinked files, uncommitted changes, etc.).  I was surprised to see warnings like "Unbrewed dylibs were found in /usr/local/lib", "Unbrewed header files were found in /usr/local/include", etc.

I'm on an M1 Mac, with homebrew properly installed in /opt/homebrew.  Why would homebrew care about any of the files in /usr/local?  I understand that Intel-based homebrew does use that folder, but I didn't think that ARM-based HB did.

Is this a mistake?  Should it be removed, or maybe changed to check in /opt/homebrew instead?

Yes, I know `brew doctor` says "these warnings are just used to help the Homebrew maintainers", but if the code is checking in the WRONG place, it's either NOT checking in the RIGHT place, or just wasting time.

**Comment by carlocab** [comment, marked as answer]
That is intentional. `brew` looks at things in `/usr/local` even when it's not installed there because `/usr/local` is in the default search path for many things. This means what is installed there is very relevant for `brew doctor`.
